### DISCUSSION TITLE
A question about `tabWidth`

### DISCUSSION THREAD

**Discussion by babakfp** [prettier/prettier#14860] [Q&A]
Hi

I set `tabWidth` to `4` because it's more readable.

As a result, I needed to override it for `.md` files. So, to prevent extra spacing between `-` and its content in list items:

```
overrides: [
    { files: "*.md", options: { tabWidth: 2 } },
],
```

My question is, how can I get code blocks inside `.md` files to get formatted to 4 tab width?

Thanks

**Comment by fisker** [comment]
Due to limitation of our internally "doc printer". It's not possible for now.
